src/ExpressiveInstaller/Resources/config/container-auryn.php

```diff
@@ -15,7 +17,7 @@
 /** @var \Auryn\Injector */
 $injector = $builder->build();
 
-$injector->share('config')->delegate('config', function () use ($config) {
+$injector->share('config')->delegate('config', function () use ($config) : ArrayObject {
```

Comment:
`ArrayObject` is not imported, so should be `\ArrayObject` or should be imported on top of the file (preferred)